### ISSUE TITLE
Optimized Installation Flow

### ISSUE THREAD

**Issue by liaoshangfeng** [aristoteleo/spateo-release#124]
Please optimize the installation experience. The existing installation method does not deal with all the dependent packages, and needs to be installed one by one, which is quite a waste of time.

**Comment by Lioscro** [comment]
Hi, @liaofeynman,
Would you please elaborate? Installing directly from Github using [these instructions](https://spateo-release.readthedocs.io/en/latest/installation.html#github) should also install all the required dependencies (except the deep-learning based cell segmentation packages).

We've validated that the dependencies are installed automatically by using a continuous integration pipeline on Github actions.

**Comment by liaofeynman** [comment]
Thank you very much for your reply. I have installed spateo in conda environment following the tutorial.
I used `python setup.py install` to install it before, which may have caused many package dependencies to be unresolved.
I noticed that there is a [tutorial](https://spateo-release.readthedocs.io/en/latest/installation.html#github) for spateo on github and it is recommended to provide the relevant link in the README.md.
